### ISSUE TITLE
Check and make changes

### ISSUE THREAD

**Issue by pottekkat** [learningOrchestra/learningOrchestra.github.io#1]
@riibeirogabriel Please check the website and make changes. Add the custom domain to this repo max early and we will start promotion.

**Comment by learningOrchestraAccount** [comment]
@navendu-pottekkat I have thinked to wait the readme modification to start the promotion, what you think? I did search the learningorchestra.com domain but doesn't available, i finded learnorchesta.com, and it is available, what you think? Can you suggest other domain name? or  learnorchesta.com sounds good?

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat I have thinked to wait the readme modification to start the promotion, what you think? I did search the learningorchestra.com domain but it doesn't available, i finded learnorchesta.com, and it is available, what you think? Can you suggest other domain name? or learnorchesta.com sounds good?

**Comment by riibeirogabriel** [comment]
Then maybe we can rename this repo to learningOrchestra-site.

**Comment by riibeirogabriel** [comment]
I saw the website, sounds very good, maybe we can wait the readme modification to update the short description of learningOrchestra.

**Comment by riibeirogabriel** [comment]
and thanks to make this awesome work!

**Comment by pottekkat** [comment]
@riibeirogabriel The .org domain name is available, since this is a free and open source project, org would also work. 

We will finish the docs first and finalise everything. And then I will make the changes to the web page and then we can plan the release

**Comment by riibeirogabriel** [comment]
Okay, i will buy yet today, then we pass the site to this domain and the docs to docs.learningorchestra.org, it will be good!

**Comment by pottekkat** [comment]
Yes, we will do that. We will change this repo name accordingly. Let me know when the docs are finished and I will make changes to the website

**Comment by riibeirogabriel** [comment]
Okay!

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat i did find learning-orchestra.com, can be better than learningorchestra.org?

**Comment by pottekkat** [comment]
Looks good! Whatever works best for you!

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat i saw some articles, and seems possible get a github.io domain without repo name after the domain, then maybe we can get a domain like learningorchestra.github.io, what do you think? to me sounds good.

**Comment by pottekkat** [comment]
Looks good! That would save getting the domain name. Whatever best would do

**Comment by riibeirogabriel** [comment]
Take a look https://learningorchestra.github.io/

**Comment by riibeirogabriel** [comment]
We can rename the learningOrchestra-docs repo to docs, then the url can be https://learningorchestra.github.io/docs

**Comment by pottekkat** [comment]
Looks good!

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat I did update the website and docs links, to me all sounds good now, please take a look in repos, if all seems good to you also, i think we can begin the learningOrchestra promotion.

**Comment by pottekkat** [comment]
I will check it out and update

**Comment by pottekkat** [comment]
@riibeirogabriel README looks good, should I change the header image? What should the new tagline be?

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat I did talk with my advisors, we think that the main feature of learningOrchestra and the most painful to build was the model builder microservice, which uses a spark cluster(distributed processing) to train and evaluate model from different classifiers setted by user, then, the currrent slogan sounds good.

**Comment by pottekkat** [comment]
@riibeirogabriel Okay then, so everything looks good right? We can plan on the release?

**Comment by riibeirogabriel** [comment]
I think so, we can.

**Comment by pottekkat** [comment]
@riibeirogabriel We can discuss in [this issue](https://github.com/learningOrchestra/learningOrchestra/issues/79).
